ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -5,54 +5,90 @@ Name of package is a name of model which was deployed.
 
 ## Structure of generated package:
 
-openvino.zip:
-
-  * [model](../model)
-    * [model.xml](../model/model.xml)
-    * [model.bin](../model/model.bin)
-  * [python](.)
-    * [README.md](./README.md)
-    * [demo.py](./demo.py)
-    * [requirements.txt](./requirements.txt)
-    * <model_name>-0.0-py3-none-any.whl
+* [model](../model)
+  * [model.xml](../model/model.xml)
+  * [model.bin](../model/model.bin)
+* [python](.)
+  * [README.md](./README.md)
+  * [demo.py](./demo.py)
+  * [requirements.txt](./requirements.txt)
+  * [demo_package-0.0-py3-none-any.whl](./demo_package-0.0-py3-none-any.whl)
 
 
 ## Prerequisites
-* Python 3.6+
+* Python 3.8+
 
 ## Setup Demo Package
 
-1. Install Python (version 3.6 or higher).
+1. Install Python (version 3.8 or higher).
 
 2. Install the package in the clean environment:
 ```
-python -m pip install <model_name>-0.0-py3-none-any.whl
+python -m pip install demo_package-0.0-py3-none-any.whl
 ```
 
 
 When the package is installed, you can import it as follows:
 ```
-python -c "from <model_name> import create_model"
+python -c "from demo_package import create_model"
 ```
 
 > **NOTE**: On Linux and macOS, you may need to type `python3` instead of `python`. You may also need to [install pip](https://pip.pypa.io/en/stable/installation/).
 > For example, on Ubuntu execute the following command to get pip installed: `sudo apt install python3-pip`.
 
 ## Usecases
 
-1. You can run demo, using entry-point <model_name>. For example if you model name `atss`, running the application with the -h option yields the following usage message:
+1. Running the `demo.py` application with the `-h` option yields the following usage message:
    ```
-   usage: atss [-h] -i INPUT -m MODEL
+   usage: demo.py [-h] -i INPUT -m MODEL [-c CONFIG]
 
    Options:
      -h, --help            Show this help message and exit.
      -i INPUT, --input INPUT
                            Required. An input to process. The input must be a
-                           single image.
+                           single image, a folder of images, video file or camera
+                           id.
      -m MODEL, --model MODEL
                            Required. Path to an .xml file with a trained model.
+     -c CONFIG, --config CONFIG
+                           Optional. Path to an .json file with parameters for
+                           model.
+
    ```
 
-   As a model, you can use `model.xml` from generated zip.
+   As a model, you can use `model.xml` from generated zip. So can use the following command to do inference with a pre-trained model:
+   ```
+   python3 demo.py \
+     -i <path_to_video>/inputVideo.mp4 \
+     -m <path_to_model>/model.xml
+   ```
 
-2. You can create your own demo application, using <model_name> package.
\ No newline at end of file
+   Also you can define own json config that specify some model parameters. To create this config please see `config.json` in demo_package wheel.
```

Comment:
If you want to expose that kind of parameters, then you should at least place a warning here, that default configuration is guaranteed to be correct, while any change should be made with caution.